pages/about/[slug].tsx

```diff
@@ -0,0 +1,92 @@
+import { useRouter } from "next/router"
+import Image from "next/legacy/image"
+import { FormattedMessage, useIntl } from 'react-intl'
+import bios from "../../data/bio"
+import board from "../../data/board"
+import { withDefaultStaticProps } from "../../utils/defaultStaticProps"
+import Layout from "../../components/Layout"
+import LogoWhite from "../../public/logos/logo-white.svg?inline"
+
+const missingAvatarSrc = require("../../public/avatars/missing_avatar.png");
+
+function mapMastodonUrlToHandle(mastodonUrl: string) {
```

Comment:
Maybe it would be easier the other way around, having the handle in the `data`, and computing the URL here?

Reply:
Yep, will update.

Reply:
Sound note that the value assigned to `socials.mastodon` is the same as what's in the `data/team.ts` file. Wanted be consistent https://github.com/mastodon/joinmastodon/blob/main/data/team.ts#L25. Also, the Mastodon link is used in the about page. A possible alternative to the link and handle is just to express the handle and domain separately. For example: 

```
{
  domain: "mastodon.social",
  handle: "mellifluousbox"
}
```

It would remove the need to split the string into parts. Just have a function to create a link and another to create a full handle.

Reply:
Spoke with @renchap in a separate channel. Agreed to keep the logic as is